index.js

```diff
@@ -0,0 +1,74 @@
+const task = document.getElementById("add-task");
+const list = document.getElementById("list");
+const clearItemsbtn = document.querySelector(".clear-btn");
+
+var editFlag = false;
+var editElement;
+
+task.addEventListener("keypress", (e) => {
+    if (e.key == "Enter") {
+        addItemToList(task.value);
+        task.value = "";
```

Comment:
same here check for the list item properly create a validate function first validate the item than add the task